package/src/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts

```diff
@@ -21,26 +21,33 @@ export const getChannelPreviewDisplayName = <
   channelName?: string;
   currentUserId?: string;
   members?: Channel<StreamChatGenerics>['state']['members'];
-}) => {
+}): string => {
   if (channelName) return channelName;
 
   const channelMembers = Object.values(members || {});
   const otherMembers = channelMembers.filter((member) => member.user?.id !== currentUserId);
 
   const name = otherMembers.slice(0).reduce((returnString, currentMember, index, originalArray) => {
     const returnStringLength = returnString.length;
-    const currentMemberName = currentMember.user?.name || currentMember.user?.id || 'Unknown User';
-    // a rough approximation of when the +Number shows up
-    if (returnStringLength + (currentMemberName.length + 2) < maxCharacterLength) {
-      if (returnStringLength) {
-        returnString += `, ${currentMemberName}`;
-      } else {
-        returnString = currentMemberName;
-      }
-    } else {
+    const currentMemberName: string =
+      currentMember.user?.name || currentMember.user?.id || 'Unknown User';
+
+    const currentNamelessThanMax: boolean =
```

Comment:
Can we found a meaningful name for this var?

Reply:
I wanna read the name and understand directly what it does

Reply:
This variable does not exist any longer in my latest commit. I think there maybe a small issue with github